docs/source/contributing.rst

```diff
@@ -76,12 +76,14 @@ Create a `conda <https://docs.conda.io>`_ environment for echopype development
 
 .. code-block:: bash
 
-    conda create -c conda-forge -n echopype --yes python=3.9 --file requirements.txt --file requirements-dev.txt
+    conda create -c conda-forge -n echopype --yes python=3.9 --file requirements-dev.txt
     conda activate echopype
     # ipykernel is recommended, in order to use with JupyterLab and IPython
     # to aid with development. We recommend you install JupyterLab separately
     conda install -c conda-forge ipykernel
-    pip install -e .
+    # plot is an extra set of requirements that can be used for plotting.
+    # the command will install all the dependencies along with plotting dependencies.
+    pip install -e .[plot]
```

Comment:
@leewujung and @emiliom I added comment here about `[plot]` let me know if it makes sense.

Reply:
Thanks. Yeah, the development environment should include the plot dependencies. Your comment looks good

Reply:
Thanks @lsetiawan ! this looks good.